### PR TITLE
Fix code blocks of all the docs

### PR DESCRIPTION
**Pull request by hs-hritik**
- Fix code blocks on the unity doc
- Fix code blocks on the ios doc
- Fix code blocks on the android doc
- Fix code blocks on the cocos2d-x doc
- Fix code blocks on the webchat doc
- Fix code blocks on the Feedback+ SDK doc
- Fix code blocks on the sdkx-unity docs
- Fix code blocks on the xamarin docs

### CODE REVIEW

**Comment by hs-hritik** [comment]
I have fixed the code blocks on Xamarin as well, but I have a few doubts on some of the places, once that I resolved, I'll push the fix for that as well.

**Review by jaydeep-helpshift — APPROVED** [review]

File: docs/xamarin/upgrade-23x-android.mdx [hunk 1]
```diff
@@ -40,21 +40,21 @@ The Helpshift SDK v2.4.0 is a major update. Please note the we have deprecated s
 
 On SDK version 2.3.1 or below, you can check if an active conversation exists as shown below:
 
-```
+```java
```
Comment (jaydeep-helpshift) [inline docs/xamarin/upgrade-23x-android.mdx:43]: csharp. In fact, all code in this file is csharp, not java

File: docs/xamarin/users-android.mdx [hunk 2]
```diff
@@ -99,7 +99,7 @@ You should call Helpshift SDK's login API whenever the user successfully logs in
   </tr>
 </table>
 
-```csharp
```
Comment (jaydeep-helpshift) [inline docs/xamarin/users-android.mdx:102]: csharp is correct. Applies to all code in this file.

File: docs/xamarin/users-ios.mdx [hunk 3]
```diff
@@ -56,7 +56,7 @@ You should call Helpshift SDK's Login API whenever the user successfully logs in
 
 For example:
 
-```
+```java
```
Comment (jaydeep-helpshift) [inline docs/xamarin/users-ios.mdx:59]: All code in this file is csharp

**Review by jaydeep-helpshift — CHANGES_REQUESTED** [review]

File: docs/xamarin/upgrade-23x-android.mdx (hunk 1 above)
Comment (hs-hritik) [reply docs/xamarin/upgrade-23x-android.mdx:43]: Ack

File: docs/xamarin/users-android.mdx (hunk 2 above)
Comment (hs-hritik) [reply docs/xamarin/users-android.mdx:102]: Ack

**Review by helpshift-tanuj — APPROVED** [review]

**Comment by hs-hritik** [comment]
Got approval from @helpshift-tanuj and @jaydeep-helpshift, I'm merging the change now. Thanks for the quick review.

**Pull request merged**